server.ts

```diff
@@ -322,10 +322,10 @@ bandwidthRtc.onParticipantLeft(async (event: ParticipantLeftEvent) => {
   const participantId = event.participantId;
   const conference = conferences.get(conferenceId);
   if (conference) {
-    // Remove the participant from our local state
-    conference.participants.delete(participantId);
     // Remove the participant from the conference
     await bandwidthRtc.removeParticipant(conferenceId, participantId);
+    // Remove the participant from our local state
```

Comment:
I'm having trouble understanding -- how does moving this keep us from calling end conference multiple times?